### PR TITLE
NXP-24576: fix selection toolbar on Safari

### PR DESCRIPTION
**Pull request by adilioaraujo**

### CODE REVIEW

**Comment by nelsonsilva** [comment]
👍

File: elements/nuxeo-app.html [hunk 1]
```diff
@@ -90,6 +90,12 @@
         height: calc(100% - var(--nuxeo-app-top));
       }
 
+      paper-header-panel {
+        --paper-header-panel-container: {
+          -webkit-overflow-scrolling: unset; /* NXP-24576: fix for selection toolbar on Safari */
```
Comment (afjusto) [inline elements/nuxeo-app.html]: on IOS actually

Comment (adilioaraujo) [reply elements/nuxeo-app.html]: Changed the commit message and comment

**Comment by afjusto** [comment]
👍

**Pull request merged**